src/Common/src/Interop/Windows/mincore/Interop.IsEqualDomainSid.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class mincore
+    {
+        [DllImport(Interop.Libraries.SecurityBase, EntryPoint = "EqualDomainSid", SetLastError = true, CharSet = CharSet.Unicode)]
+        internal static extern int IsEqualDomainSid(byte[] sid1, byte[] sid2, out bool result);
```

Comment:
I went from Security to SecurityBase because "api-ms-win-security-base-l1-1-0.dll" is already in the OSS tree with this name.  Whoever made that decision is a great person to ask... I do not really know but it would involve either duplication or touching lots of other files to fix.